pkg/utils/eden.go

```diff
@@ -10,17 +10,91 @@ import (
 )
 
 const (
-	adamContainerName = "eden_adam"
-	adamContainerRef  = "lfedge/adam"
-	logLevelToPrint   = log.InfoLevel
+	redisContainerName = "eden_redis"
+	redisContainerRef  = "redis"
+	adamContainerName  = "eden_adam"
+	adamContainerRef   = "lfedge/adam"
+	logLevelToPrint    = log.InfoLevel
 )
 
+//StartRedis function run redis in docker with mounted redisPath:/data
+//if redisForce is set, it recreates container
+func StartRedis(redisPort string, redisPath string, redisForce bool, redisTag string) (err error) {
```

Comment:
Can we make some other file with StartRedis/StopRedis functions? E.g. pkg/utils/redis.go

Reply:
OK -- we'll think about it.